src/qiskit_iqm/fake_backends/quantum_architectures/adonis.py

```diff
@@ -19,11 +19,26 @@
 
 
 class Adonis(IQMQuantumArchitecture):
-    """Class implementation for IQMs 5-qubit QPU architecture."""
+    """IQM's five-qubit transmon device.
+
+    The qubits are connected thus::
+            QB1
+             |
+      QB2 - QB3 - QB4
+             |
+            QB5
+
+    where the lines denote which qubit pairs can be subject to two-qubit gates.
+
+    Each qubit can be rotated about any axis in the xy plane by an arbitrary angle.
+    Adonis thus has native PhasedXPowGate, XPowGate, and YPowGate gates. The two-qubit gate CZ is
```

Comment:
PhasedXPowGate, XPowGate, and YPowGate gates are gates in Cirq. Adonis' native single-qubit gate is Phased Rx, equivalent to https://qiskit.org/documentation/stubs/qiskit.circuit.library.RGate.html in Qiskit.